scripts/cluster-upgrade.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Variable sanity check
+if [[ -z ${PGVERSIONOLD} ]]; then
+  echo "Environment variable PGVERSIONOLD is empty."
+  echo "It must be set to postgresql version of the old cluster."
+fi
+if [[ -z ${PGVERSIONNEW} ]]; then
+  echo "Environment variable PGVERSIONNEW is empty."
+  echo "It must be set to postgresql version of the new cluster."
+fi
+if [[ -z ${PGDATAOLD} ]]; then
+  echo "Environment variable PGDATAOLD is empty."
+  echo "It must be set to the location of the old cluster."
+fi
+
+if [[ -z ${PGUNIXUSEROLD} ]]; then
+  PGUNIXUSEROLD=postgres
+fi
```

Comment:
Does this ever change ?

Reply:
It might be if it comes from different image and you want to preserve the ownership so the same datadir can be used in previous image. 
But the option is not mandatory